src/lib/support/tests/TestScopedBuffer.cpp

```diff
@@ -0,0 +1,160 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "TestSupport.h"
+
+#include <support/ScopedBuffer.h>
+#include <support/TestUtils.h>
+
+#include <nlunit-test.h>
+
+using namespace chip;
```

Comment:
This should obviate the need for the `chip::` in various places in this file.

Reply:
Dropped the line. we don't have too many chip:: remaining so this should be fine. Somehow Platform cannot be without chip:: prefix so the worst offender cannot be removed.

Reply:
Actually rechecked and in this context I could remove chip:: and keep just Platform. Still unsure if worth it.